scripts/gis-material-update/config.yaml

```diff
@@ -28,4 +40,26 @@ polkupyora_paareitti:
 liikennevaylat:
 
 # manuaalisesti ladattavat
-kuntarajat:
\ No newline at end of file
+kuntarajat:
+
+database:
+  local_development:
+    port: 15432
+    host: localhost
+    username: haitaton
+    password: haitaton
+    database: haitaton
+  local_docker_development:
+    port: 5432
+    host: gis-db
+    username: haitaton
+    password: haitaton
+    database: haitaton
```

Comment:
Instead of grouping by value, like
```
key_a:
  env_a:
    ...
  env_b:
    ...

key_b:
  env_a:
    ...
  env_b:
    ...
```
Would these be better when grouped by environment?
```
env_a:
  key_a:
    ...
  key_b:
    ...
env_b:
  key_a:
    ...
  key_b:
    ...
```
This way, it's easier to see the values of a single environment.

The different environments could even be split into different files, like `config.local_development.yaml`, but then there would need to be logic for reading default values from `config.yaml`, since many values don't change between environments.

Reply:
On the other hand, when considering running this on the cloud at some point, we don't want to change the repository every time we need to change some configuration value in a cloud environment. So there should be a method to override these values from environment variables. Then we wouldn't need the different environments in this file at all. This could just hold the local development settings and config for the local docker development could be set as environment variables in the docker compose config.